stable/artifactory/templates/nginx-deployment.yaml

```diff
@@ -66,9 +68,7 @@ spec:
               - '/bin/sh'
               - '-c'
               - >
-                {{- if .Values.nginx.customArtifactoryConfigMap }}
-                cp -Lrf /tmp/conf.d/artifactory*.conf /etc/nginx/conf.d/artifactory.conf;
```

Comment:
How would this get copied not to /etc/nginx?

Reply:
Directly mounting it there

Reply:
It’s still mounted to /tmp